core/domain/classifier_services.py

```diff
@@ -434,6 +474,9 @@ def load_examples(self, examples):
         """Sets the internal state of the classifier, assigns random initial
         labels to the docs, and runs Gibbs sampling for _training_iterations
         number of iterations.
+
+        Args:
+            examples: A list of examples (see class docstring for example).
```

Comment:
I simply copied part of the example in the class docstring, this one has the same signature as **add_examples_for_training**.